qip/examples/deutch.rs

```diff
@@ -0,0 +1,73 @@
+#[cfg(feature = "macros")]
+use qip::builder::Qudit;
+#[cfg(feature = "macros")]
+use qip::builder_traits::UnitaryBuilder;
+#[cfg(feature = "macros")]
+use qip::prelude::*;
+#[cfg(feature = "macros")]
```

Comment:
This was a mistake in the `macro_example` but is correct in `inverse_example` -- you only need the `#[cfg(...)]` above the `use qip_macros::program;` since the others all exist even if the `macros` feature isn't being used.

Reply:
Got it. Thank you for the feedback!